### PR TITLE
feature/EMDI-211_SwiftUI_Emerald_checkbox

### PR DESCRIPTION
**Pull request by dpolania91**

### CODE REVIEW

**Review by lborrego037 — DISMISSED** [review]

File: EmeraldIOS.xcodeproj/project.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "self:CondorUIComponents.xcodeproj">
```
Comment (Cristophere96) [inline EmeraldIOS.xcodeproj/project.xcworkspace/contents.xcworkspacedata:5]: Are you sure this should be deleted?

Comment (dpolania91) [reply EmeraldIOS.xcodeproj/project.xcworkspace/contents.xcworkspacedata:5]: this line was deleted when cleaning the project, I tried again and it did not generate any error.

File: EmeraldIOS/Components/SwiftUi/SwiftUICheckbox/EmeraldSwiftUICheckbox.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    }
+    
+    public init(checked: Binding<Bool>, text: String) {
+        _checked = checked
+        self.text = text
+    }
+    
+    public var body: some View {
+        HStack {
+            Image(systemName: checked ? "checkmark.square.fill" : "square")
```
Comment (Cristophere96) [inline EmeraldIOS/Components/SwiftUi/SwiftUICheckbox/EmeraldSwiftUICheckbox.swift:30]: Place this icons names within the `InnerConstant`

File: EmeraldIOS/Components/SwiftUi/SwiftUICheckbox/EmeraldSwiftUICheckbox.swift [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+//
+
+import SwiftUI
+
+public struct EmeraldSwiftUICheckbox: View {
+    @Binding var checked: Bool
+    var text: String
+    
+    private struct InnerConstant {
+        static let twenty = 12
```
Comment (Cristophere96) [inline EmeraldIOS/Components/SwiftUi/SwiftUICheckbox/EmeraldSwiftUICheckbox.swift:16]: Better create this constants as `CGFloat` so you avoid creating one when using them

**Review by Cristophere96 — CHANGES_REQUESTED** [review]

**Review by Cristophere96 — APPROVED** [review]

**Review by lborrego037 — APPROVED** [review]

**Pull request merged**